app/assets/stylesheets/courses.css

```diff
@@ -0,0 +1,33 @@
+.courses-section {
+  display: flex;
+  flex-direction: column;
+  gap: 2.5rem;
+}
+
+.all-courses {
+  align-self: center;
+  margin-top: 1rem;
+}
+
+.single-course {
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 0.8rem;
+  text-align: center;
+  border: 1px solid red;
```

Comment:
I think we don't need borders in course items. Kindly remove this style.

Reply:
@IndieCoderMM Thanks for identifying that, I forgot to remove the borders